source/reference/failure-recovery.txt

```diff
@@ -12,22 +12,24 @@ Recovery From Sync Failure
    :depth: 2
    :class: singlecol
 
+Disaster Recovery is a use case of {+c2c-product-name+}.  Synchronization can
```

Comment:
[suggestion] I'd like to see a more descriptive intro sentence here. Perhaps setting up `mongosync` for disaster recovery is done the same way as any other connection? (Let's verify this.) If so, we could start with that. Something like, 

"You can use `mongosync` as a disaster recovery utility in the event of source cluster failure. [Connect your clusters](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/connecting/) and return to this page for instructions in the event you need disaster recovery."

Reply:
Fixed